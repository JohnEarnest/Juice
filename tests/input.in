34 19 12 18