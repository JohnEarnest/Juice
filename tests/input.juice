// input test

int sum   = 0;
int count = 0;
while(hasNextInt()) {
	sum   = sum   + nextInt();
	count = count + 1;
}

print("sum: ");
print(sum);
println();

print("count: ");
print(count);
println();

if (count > 0) {
	print("average: ");
	print(sum / count);
	println();
}