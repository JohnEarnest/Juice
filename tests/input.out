sum: 83
count: 4
average: 20
